### PR TITLE
Added code to get direct counter stats via GET CLI command

### PR DESCRIPTION
**Pull request by bharticemk**

### CODE REVIEW

File: clients/p4rt-ctl/p4rt-ctl.in [hunk 1]
```diff
@@ -1677,6 +1678,42 @@ def parse_rcflow(flow):
...
 
+def _format_dce(ce):
+    output_buffer = 'table_id={}'.format(ce.table_entry.table_id)
+    output_buffer += ', counter_data=(bytes={},packets={})'.format(ce.data.byte_count, ce.data.packet_count)
+    return output_buffer
+
+@with_client
+def p4ctl_get_direct_counter_entry(client, bridge, cnt_tbl_name, flow):
+    """
+    get-direct-counter-entry SWITCH COUNTER_TABLE READ_COUNTER_FLOW
```
Comment (nupuruttarwar) [inline clients/p4rt-ctl/p4rt-ctl.in:1689]: in the format on line 1689, it says **get-direct-counter-entry** but in example on line 1691, it says **get-direct-counter**

File: clients/p4rt-ctl/p4rt-ctl.in [hunk 2]
```diff
@@ -1677,6 +1678,42 @@ def parse_rcflow(flow):
...
+         match_fields=key
+     )
+    ce.table_entry.CopyFrom(te)
+    entity.direct_counter_entry.table_entry.CopyFrom(te)
+    for response in client.read_one(entity):
+        for entry in response.entities:
+            if entity.direct_counter_entry.table_entry.table_id == ce.table_entry.table_id:
+                print(_format_dce(entry.direct_counter_entry))
+
+
```
Comment (nupuruttarwar) [inline clients/p4rt-ctl/p4rt-ctl.in:1715]: 3 blank lines?

File: clients/p4rt-ctl/p4rt-ctl.in (hunk 1 above)
Comment (bharticemk) [reply clients/p4rt-ctl/p4rt-ctl.in:1689]: line 1689 is added just as comments wrt to function name p4ctl_get_direct_counter_entry as get-direct-counter-entry
and "get-direct-counter" is CLI command

File: clients/p4rt-ctl/p4rt-ctl.in (hunk 2 above)
Comment (bharticemk) [reply clients/p4rt-ctl/p4rt-ctl.in]: Fixed

File: clients/p4rt-ctl/p4rt-ctl.in [hunk 3]
```diff
@@ -74,6 +74,7 @@ for P4Runtime switches:
   get-action-profile-group SWITCH PROFILE FLOW  prints group entries
 
   get-counter SWITCH CTR_TBL CTR_FLOW   gets indirect counter table entry
+  get-direct-counter SWITCH CTR_TBL CTR_FLOW   gets direct counter table entry
```
Comment (5abeel) [inline clients/p4rt-ctl/p4rt-ctl.in:77]: Do we need a set/reset counter in addition to get-counter? What is the requirement here?

Comment (bharticemk) [reply clients/p4rt-ctl/p4rt-ctl.in:77]: the requirement was to read the direct counter stats for the traffic wrt configured rule in FxP

**Review by n-sandeep — APPROVED** [review]
LGTM

**Review by nupuruttarwar — APPROVED** [review]
LGTM

**Pull request merged**